supadata/youtube.py

```
@@ -183,6 +188,27 @@ def playlist(self):
             self._playlist_instance = self._Playlist(self)
         return self._playlist_instance
 
+    @property
+    def video(self):
+        """Video namespace for YouTube operations."""
+        if self._video_instance is None:
+            self._video_instance = self._Video(self)
+        return self._video_instance
+
+    @property
+    def transcript(self):
```

Ruff (0.8.2)

Redefinition of unused `transcript` from line 36

(F811)